feedback/githubService.js

```diff
@@ -0,0 +1,23 @@
+export async function createGitHubIssue(label, title, body) {
```

Comment:
Some documentation here for sure.  A link to the GH docs where you figured out how to do this would be nice.

Reply:
nah, not for something we are going to throw away. This isn't really intended for contribution or extension